static/app/views/partials/sidebar.html

```diff
@@ -0,0 +1,18 @@
+<ul class="nav nav-sidebar">
+  <ul class="nav navbar-nav">
+      <li class="dropdown">
```

Comment:
Sidebar looks great and it does make it easier to navigate. I'm assuming one of the org dropdown will be removed later.